pkg/server/server.go

```diff
@@ -1362,9 +1366,6 @@ func (s *Server) IsReady(ctx context.Context) (bool, error) {
 // resolveTypesystem resolves the underlying TypeSystem given the storeID and modelID and
 // it sets some response metadata based on the model resolution.
 func (s *Server) resolveTypesystem(ctx context.Context, storeID, modelID string) (*typesystem.TypeSystem, error) {
-	ctx, span := tracer.Start(ctx, "resolveTypesystem")
```

Comment:
this span is pretty useless. it's not logging the store id. i can't do anything with just the model_id. so i'm removing it and putting all the info within the `typesystemResolver` span.